internal/infrastructure/kubernetes/deployment.go

```diff
@@ -258,5 +258,5 @@ func EnvoyPodSelector() *metav1.LabelSelector {
 
 // envoyLabels returns the labels used for Envoy.
 func envoyLabels() map[string]string {
-	return map[string]string{"control-plane": "envoy-gateway"}
+	return map[string]string{"app": "envoy"}
```

Comment:
thoughts on `app : envoyproxy` for DP and `app: envoygateway` for CP ?

Reply:
@arkodg @zirain both of you provide viable approaches to labeling. Since we are scheduled to release RC1 tomorrow and this fixes a blocking bug, I created https://github.com/envoyproxy/gateway/issues/290 to establish label conventions after the release.